datasets/gov/vstt/saugomu_ter_kadastras.csv

```diff
@@ -0,0 +1,718 @@
+,,,,,spec_reglam,string,,,,4,open,,Specialieji / individualieji veiklos reglamentai,
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai (nurodomi moksliniai tyrimai ir publikacijos),
+,,,,,tarpt_svarba_status,integer,,,,4,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
+,,,,,tarpt_svarba_apras,string,,,,3,open,,Aprašas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statuso aprašymas
+,,,,,duomen_monitor,string,,,,2,open,,Monitoringas,Aplinkos stebėsenos (monitoringo) duomenys
+,,,,,pastabos,string,,,,2,open,,Pastabos,
+,,,,,id_vart,integer,,,,2,open,,Duomenų teikėjų klasifikatorius,
+,,,,,reiksme,integer,,,,2,open,,Objekto reikšmė,
+,,,,,steigejas_galiojantis,string,,,,2,open,,Steigėjas,"Institucija, kurios potvarkiu šiuo metu galioja saugoma teritorija arba jos riba"
+,,,,,steig_data_galiojantis,date,D,,,4,open,,Šiuo metu galiojančio saugomos teritorijos steigimo teisės akto data,
+,,,,,steig_ta_galiojantis,string,,,,4,open,,Teisinis statusas,"Teisės aktas, pagal kurį šiuo metu galioja saugomos teritorijos riba, reglamentai arba šis teisės aktas yra šiuo metu galiojantis saugomos teritorijos teisinis pagrindas"
+,,,,,tvarkymo_zona_apras,string,,,,2,open,,Duomenys apie kraštovaizdžio tvarkymo zonas,
+,,,,,funkc_prior_zon_apras,string,,,,4,open,,Duomenys apie biosferos poligono funkcinio prioriteto zonas,
+,,,,,planavimo_dok_apras,string,,,,2,open,,Duomenys apie parengtus planavimo dokumentus,
+,,,,,vieta,string,,,,2,open,,Vieta,"Vieta, aprašomoji informacija (savivaldybė, seniūnija, jei miškas – papildomai miškų urėdija, girininkija, miško kvartalas ir sklypas)"
+,,,,,geometrija,geometry,,,,4,open,,Ploto geometrija,
+,,,,,,comment,type,,"update(type: ""geometry(linestring, 3346)"")",3,open,,,
+,,,,,,,,,,,,,,
+,,,,BiosferosRezervatas,,,id,,,4,,,Biosferos rezervatai,
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,pavadinimas,string,,,,4,open,,Biosferos rezervato pavadinimas,
+,,,,,gkodas,string,,,,2,open,,InGIS gkodas (saugomos teritorijos tipas),
+,,,,,plotas,number,ha,,,4,open,,"Plotas, ha",
+,,,,,steigejas,string,,,,2,open,,Steigėjas – institucija,
+,,,,,steig_data,datetime,D,,,4,open,,"Steigimo data (metai, mėnuo, diena)",
+,,,,,steig_ta,string,,,,4,open,,Įsteigimo teisės aktas,
+,,,,,steig_tikslas,string,,,,2,open,,Steigimo tikslas (aprašomoji informacija),
+,,,,,bendr_reglam,integer,,,,4,open,,Bendrieji veiklos reglamentai,
+,,,,,spec_reglam,string,,,,4,open,,Specialieji / individualieji veiklos reglamentai,
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai (nurodomi moksliniai tyrimai ir publikacijos),
+,,,,,tarpt_svarba_status,integer,,,,4,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
```

Comment:
```suggestion
,,,,,tarpt_svarba_status,integer,,,,2,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
```
Duomenys pateikti kodais, nėra atskirai pateiktos duomenų reikšmės ar klasifikatorius.
https://atviriduomenys.readthedocs.io/dsa/dimensijos.html#enum

Jau buvo pateiktas šis siūlymas `model=BiosferosPoligonas`, buvo rekomenduota atitinkamai patikslinti visame ADSA.